storage-kit/src/p2p/actor.rs

```diff
@@ -0,0 +1,122 @@
+use anyhow::{anyhow, Result};
+use tokio::{select, sync::{mpsc::Receiver, oneshot}};
+use std::time::Duration;
+use libp2p::{futures::StreamExt, identify, identity::Keypair, noise, swarm::NetworkBehaviour, Multiaddr, Swarm};
+use tracing::*;
+
+use crate::{Actor, ActorEventLoop, Port};
+
+/// Defines max_negotiating_inbound_streams constant for the swarm.
+/// It must be set for large plots.
+const SWARM_MAX_NEGOTIATING_INBOUND_STREAMS: usize = 100000;
```

Comment:
_(question from author)_

Should we centralize **all** the constants in 1 file?

Reply:
I prefer the latter actually.